partcad/src/partcad/__init__.py

```
@@ -1,7 +1,6 @@
 from build123d import Location
-import sentry_sdk
-from sentry_sdk.integrations.logging import LoggingIntegration
 
+from .sentry import init_sentry, instrument, tracer
```

Ruff (0.8.2)

`.sentry.instrument` imported but unused; consider removing, adding to `__all__`, or using a redundant alias

(F401)